Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,72 @@
+# Microsoft Mixed Reality Toolkit Release Notes
+
+- [Version 2.0.0](#version-200)
+
+## Version 2.0.0
+
+- [What's new](#whats-new-in-200)
+- [Known issues](#known-issues-in-200)
+- [Changes](#changes-in-200)
+
+This release of the Microsoft Mixed Reality Toolkit 2.0.0 supports the following devices and platforms.
+
+- Microsoft HoloLens 2
+- Microsoft HoloLens (1st gen)
+- Windows Mixed Reality Immersive headsets
+- OpenVR
+
+The following software is required.
+
+- Microsoft Visual Studio (2017 or 2019) Community Edition or higher
+- Windows 10 SDK 18362 or later (installed by the Visual Studio Installer)
+- Unity 2018.4, 2019.1 or 2019.2
+
+### What's new in 2.0.0
+
+**Default HoloLens (1st gen) profile**
+
+We have added a new profile for HoloLens (1st gen) development that includes some of the
+recommended MRTK configurations for best performance.
+
+**CoreServices**
+
+The CoreServices static class works in conjunction with the MixedRealityServiceRegistry
+to provide applications with a fast and convenient mechanism to aquire instances of core
+services (ex: Input System).
```

Comment:
Consider adding a link to documentation about this

Reply:
done